### PR TITLE
[playground] Adds support for Netlify

### PR DESCRIPTION
**Pull request by joelalejandro**
### Description

This PR injects the necessary configuration for Netlify to deploy a static version of the Playground.

### CODE REVIEW

**Comment by snario** [comment]
I think you need to specify `packages/playground/dist`

File: packages/playground/.nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+10.13.0
```
Comment (snario) [inline packages/playground/.nvmrc]: I'm on 10.5.0. Why did you choose 10.13.0?

Comment (joelalejandro) [reply packages/playground/.nvmrc]: @snario 10.13.0 is the standard LTS version. We should be working on top of that.

File: packages/playground/netlify.toml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [build]
-  base    = "packages/playground"
+  base    = "."
   publish = "dist"
   command = "npm run build"
```
[the remark this answers is not in the document]
Comment (joelalejandro) [reply packages/playground/netlify.toml]: @snario Does Netlify has built-in "yarn" support? If so, I'll change it. I didn't see anywhere that it's preinstalled on the container or something.

[the remark this answers is not in the document]
Comment (joelalejandro) [reply packages/playground/netlify.toml]: @snario Actually, scratch that - I just saw in the logs it does.

**Comment by joelalejandro** [comment]
@snario About the "base" value, since it's now on `packages/playground` it shouldn't need the entire path.

File: packages/playground/.nvmrc (hunk 1 above)
Comment (snario) [reply packages/playground/.nvmrc]: Okay

**Pull request merged**